source/mainnet/net/nodes/baker-macos.rst

```diff
@@ -34,11 +34,11 @@ On mainnet
 
    .. code-block:: console
 
-      $sudo cp "/path/to/mainnet/baker-credentials.json" "/Library/Application Support/Concordium Node/Mainnet/Config/baker-credentials.json"
+      $sudo cp "/path/to/mainnet/baker-credentials.json" "/Library/Application\ Support/Concordium\ Node/Mainnet/Config/baker-credentials.json"
```

Comment:
the backslash from "Application Support" needs to be removed. It should only be part of the "Concordium Node" folder.

Reply:
Why is that?
I thought that if there should be a space in the path then backslash should stay as it is an escape character. Similar to Concordium\ Node.

Reply:
From what I noticed is that "Application Support" being an OS defined folder, it's accepted without a backslash . If we use the backslash there, the folder will not be found and another one will be created. This doesn't apply for "Concordium\Node" as it's a folder we are creating.